docs/tutorials/projects/tipsy.md

```diff
@@ -715,22 +716,24 @@ It also creates two background tasks running asynchronously, one looking for obs
 async def main():
     robot = await connect()
     base = Base.from_robot(robot, base_name)
-    sensors = [Sensor.from_robot(robot, sensor_name)
-               for sensor_name in sensor_names]
-    detector = VisionServiceClient.from_robot(robot, "myPeopleDetector")
+    sensors_svc = SensorsClient.from_robot(robot, name=sensor_service_name)
+    sensors = await sensors_svc.get_sensors()
+    detector = VisionClient.from_robot(robot, name=detector_name)
 
     # create a background task that looks for obstacles and stops the base if
-    # it's moving
+    # its moving
     obstacle_task = asyncio.create_task(
-        obstacle_detect_loop(sensors, base))
+        obstacle_detect_loop(sensors, sensors_svc, base))
     # create a background task that looks for a person and moves towards them,
     # or turns and keeps looking
     person_task = asyncio.create_task(
-        person_detect(detector, sensors, base))
+        person_detect(detector, sensors, sensors_svc, base))
     results = await asyncio.gather(obstacle_task,
                                    person_task,
                                    return_exceptions=True)
     print(results)
+
+    await robot.close()
 ```
 
 When you run the code, you should see results like this:
```

Comment:
here we have to add:
{namespace: "rdk"
type: "component"
subtype: "sensor"
name: "ultrasonic"
: {'distance': 0.7528849999999999}, namespace: "rdk"
type: "component"
subtype: "sensor"
name: "ultrasonic2"
: {'distance': 1.2973975}}

since we are printing the sensors in the code

Reply:
Or should we just remove the print? We didn't ahve it before

Reply:
ok removed per conversation